docs/blog/2018-11-06-gatsby-for-apps/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Gatsby for Apps
+date: 2018-11-06
+author: Dustin Schau
+tags: ["app", "web app", "application", "authentication"]
+---
+
+Gatsby is truly incredible for static sites. In fact, our branding for Gatsby used to be the following:
+
+> a blazing-fast static site generator for React
+
+This is _still_ true, but I'd like to clarify one, simple concept. Gatsby is _still_ great for static sites, but it's also an equally great choice for applications. In this post, I'd like to clarify this central concept, and set the record straight that Gatsby is great for static sites and applications, alike.
```

Comment:
I'm a fan of using "and" instead of "but". A personal preference since I think Gatsby is great for static sites AND it's an equally great choice for applications. They aren't mutually exclusive!